packages/@expo/config-plugins/src/android/Updates.ts

```diff
@@ -57,6 +72,19 @@ export const withUpdates: ConfigPlugin<{ expoUsername: string | null }> = (
   });
 };
 
+const withRuntimeVersionResource = createStringsXmlPlugin(applyRuntimeVersionFromConfig, 'withRuntimeVersionResource');
+
+function applyRuntimeVersionFromConfig(
+  config: Pick<ExpoConfigUpdates, 'sdkVersion' | 'runtimeVersion'>,
+  stringsJSON: ResourceXML
+): ResourceXML {
+  const runtimeVersion = getRuntimeVersionNullable(config, "android");
+  if (runtimeVersion) {
+    return setStringItem([buildResourceItem({ name: 'runtime_version', value: runtimeVersion })], stringsJSON);
```

Comment:
Maybe expo_runtime_version or somn so ppl can trace it a bit better since strings is essentially a singleton

Reply:
Done